dangerfile.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
```

Comment:
MIT license please :-)

Reply:
Ah yeah! 

Sure, (I've been hacking on this slowly over the last two days) I am a bit surprised by the build failures but I think this is pretty much good to go once those are figured

Reply:
Done 👍